### PR TITLE
Wip hyperledger fabric monitoring

### PR DESCRIPTION
**Pull request by zaouchelo**
This PR aim to add Hyperledger Fabric plugins

### CODE REVIEW

**Review by Sims24 — CHANGES_REQUESTED** [review]
H @zaouchelo 

Please clean the code and remove all dumper and commented code. 

Also, you are modifying some common mode like openmetrics one, it's a problem because it breaks existing stuff.

**Comment by Sims24** [comment]
Or maybe you can convert your PR as draft ;)

**Comment by Sims24** [comment]
@zaouchelo please provide update on the status and/or ETA of a ready to review PR. 

Also, if you can provide more privately an environment to test the Plugin it would be awesome.

**Comment by zaouchelo** [comment]
@Sims24 Sorry I was busy with other stuff... I planned to work on it early next week.
Yes, I'll share with you the test environment.

File: blockchain/hyperledger/exporter/ledger/mode/metrics.pm [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+    return sprintf(
+            "Average block processing duration (sec) : %s, Block count: %s, Transaction count: %s",
+            $self->{result_values}->{ledger_block_processing_time_avg},
+            $self->{result_values}->{ledger_blockchain_height},
+            $self->{result_values}->{ledger_transaction_count}
+    );
+}
+
+sub prefix_output_leger {
```
Comment (Sims24) [inline blockchain/hyperledger/exporter/ledger/mode/metrics.pm:64]: typo missing 'd'

**Comment by Sims24** [comment]
Won't merge. Expiring.

**Pull request closed without merging**